test_0 (__main__.lab3e.test_0)
[Lab 3] - [Investigation 3] - [Part 1] - Functions - Test for file creation: ./lab3e.py ... ok
test_a (__main__.lab3e.test_a)
[Lab 3] - [Investigation 3] - [Part 1] - Functions - Test for errors running: ./lab3e.py ... ok
test_a1 (__main__.lab3e.test_a1)
[Lab 3] - [Investigation 3] - [Part 1] - Functions - Test for correct shebang line: ./lab3X.py ... ok
test_a_function_list (__main__.lab3e.test_a_function_list)
[Lab 3] - [Investigation 3] - [Part 1] - functions & lists - print first item in list ... ok
test_b_function_first (__main__.lab3e.test_b_function_first)
[Lab 3] - [Investigation 3] - [Part 1] - functions & lists - print first item in list ... FAIL

======================================================================
FAIL: test_b_function_first (__main__.lab3e.test_b_function_first)
[Lab 3] - [Investigation 3] - [Part 1] - functions & lists - print first item in list
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/akumar250/ops-lab3/CheckLab3.py", line 412, in test_b_function_first
    self.assertEqual(expected_output, lab3eStudent.give_first_item(), msg=error_output)
AssertionError: '100' != 100 : your function must return a string value(HINT: use the str() function)

----------------------------------------------------------------------
Ran 5 tests in 0.039s

FAILED (failures=1)
